app/src/main/java/com/vultisig/wallet/ui/models/keygen/FastVaultVerificationViewModel.kt

```
@@ -93,32 +93,36 @@ internal class FastVaultVerificationViewModel @Inject constructor(
+                        delay(FAST_VAULT_VERIFICATION_SUCCESS_DELAY)
+                        navigator.route(
+                            route = Route.BackupVault(
+                                vaultId = args.vaultId,
+                                vaultType = VaultType.Fast,
+                            )
                         )
-                    }
 
-                    if (context.canAuthenticateBiometric()) {
-                        vaultPasswordRepository.savePassword(vaultId, vault.password)
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)